00-basics/002-javascript/Kristina/Learning002.js

```diff
@@ -0,0 +1,143 @@
+let RobotDinosaurFromSpace = {};
```

Comment:
Since you never assign any other value to variable `RobotDinosaurFromSpace`, you can define it as `const` (note that you can modify fields of constants just fine (`RobotDinosaurFromSpace.roar = <...>` will work), you just can't reassign constant itself (`RobotDinosaurFromSpace = <...>` will not))